myapp/app/controllers/tasks_controller.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 class TasksController < ApplicationController # rubocop:todo Style/Documentation
+  before_action :check_authentication
   def index
     @query = Task.ransack(params[:q])
-
-    @tasks = @query.result.custom_order(params[:sort]).page(params[:page])
+    puts(@current_user["id"])
```

Comment:
can't we use `current_user.id` here? as we already created current_user method in application controller

Note: I didn't run the code, its just my assumption

Reply:
Yeah we can, actually i was checking the value and end up forgetting to remove before committing. 🤣

Reply:
removed